addons/account_payment/views/account_portal_templates.xml

```diff
@@ -7,7 +7,7 @@
             <td class="text-center">
                 <t t-set="tx_ids" t-value="invoice.transaction_ids.filtered(lambda tx: tx.state in ('pending', 'authorized', 'done'))"/>
                 <t t-set="pending_manual_txs" t-value="tx_ids.filtered(lambda tx: tx.state == 'pending' and tx.acquirer_id.provider in ('transfer', 'manual'))"/>
-                <a t-if="invoice.state == 'posted' and invoice.invoice_payment_state == 'not_paid' and invoice.amount_total and invoice.type == 'out_invoice' and (pending_manual_txs or not tx_ids)"
+                <a t-if="invoice.state == 'posted' and invoice.invoice_payment_state == 'not_paid' and invoice.amount_total and invoice.type == 'out_invoice' and (pending_manual_txs or not tx_ids or invoice.amount_residual)"
```

Comment:
Your fix is wrong. Use `invoice.invoice_payment_state in ('not_paid', 'partial')` instead.

Reply:
Hello :slightly_smiling_face: Alright, will do but while re-testing the flow described in my commit, I see that the behavior is changed: at the step of adding shipping fee, I have a negative amount on my second invoice, is it correct?
![image](https://user-images.githubusercontent.com/44211228/110350809-6371f700-8034-11eb-89c7-e2f0670835d4.png)